pdns/recursor_cache.cc

```diff
@@ -15,6 +15,8 @@
 #include "cachecleaner.hh"
 #include "rec-taskqueue.hh"
 
+uint16_t MemRecursorCache::s_maxServedStaleExtensions;
```

Comment:
Hmm, so this file is no longer a symlink, but we now have two versions?

Reply:
Or is it a GH bug?

Reply:
Lemme heck that, I might have used patch(1) locally, which has the habbit of changing a symlink to a file.